Logo/svg-resources/Lidarr-Text.svg

```diff
@@ -0,0 +1,35 @@
hweOjY2l14wVicJDQEQFAoWHywfHlE2nKAkP1ZkbTc3bzU1XSQOIxQ/rmowXUsuIThMKoM7aE4tJB1CRzs2NpZeWEqiP1o5Gxs6WT8/WjkbGzlaBYoTICsZGy0fEhMhLBkZKyAT/oYCBw4NJkQkDA4JAwEtcEJgjzIyMwsOGDItJQsGBwUDAYOKOFdAKxsLDAkJFwsiRB4SIgwhNywuMhgEEitGMzFpNjCeZ16RMzMyF3MmQ103OFxDJSVDXTc3XUMmAAAAAAIAdf2bBOkFtAAlADUASUAQIwglERAQIUQlNzUZRgg2NLgBOUATKbglSCMjAxEUShANAyEeSgADCQA/M+0yPzPtMhI5L+0/7QEQ3v3MEN7tMy8zERI5MDElBgYjIiYmAjU0EjY2MzIWFwcmJiMiDgIVFB4CMzI2NxEjNSEBNjYzMhYWBgcOAwcjEwTpd/94nPGkVVem85xh1nE1ZbFadK5zOTVuqnRSpkXxAab9mCY3HDE8GwQQCyQoKQ96T00xNFi0ARG5yQEgt1YnJJYiIkSO3JiW3pFHIhcBjpf8bxsUJz1OJxpGS0gaAQYAAwBh/kkEOwahAA8AagB+AHxAQnVrcAF6UyBRUzk5GBUggHBTSUZcXGJ/GGtXaHVXenBoJiZXahVoATNXYFdwVwITVwFXB0lMV0ZgQHBAAhJAAUCvD7oBOQAFATcAP+0/XV0z7TI/XV3tPzMzEjkvEjk57RDtMgEQ3jIRMzPtEM4yMjIQ7RD9zBI5OTAxAQMXBgYjIiYmNjc+AzcTMj4CNxYWFw4DBxYWFRQGBwYGIyImJwYGFhYXHgMzFjIzMhYVFA4EIyImJyYmJzY2NxYWMzI+AjU0LgIjIyIuAjU0NjcmJjU0Njc2NjMyFwciDgIVFB4CMzI+AjU0LgICp08zJjccMTwbBBAKJSgpD80rQTw9KQ0ZDSA2NTUeHB46NjaXXjBWJwkNARAUChYfLB8eUTacoCQ/VmRtNzdvNTVdJA4jFD+uajBdSy4hOEwqgztoTi0kHUJHOzY2ll5YSqI/WjkbGzpZPz9aORsbOVoGof76sRsUJz1OJxlHS0ga/W8CBw4NJkQkDA4JAwEtcEJgjzIyMwsOGDItJQsGBwUDAYOKOFdAKxsLDAkJFwsiRB4SIgwhNywuMhgEEitGMzFpNjCeZ16RMzMyF3MmQ103OFxDJSVDXTc3XUMmAAACAGQAAAZSBZ4AEwAXAEpAJhcJCRFEEBgUCAgAGQFEAA8CEsoWDAUWF0gWEAgIBxACCwgHCAACAD8/Pz8SOS8SOe0RMzMQ7TIyAS/9xBEzETMQ1O0yETMwMQEzFTMVIxEjESERIxEjNTM1MxUhETUhFQTmvLCwvPz7vMHBvAMF/PsFnvB5+8sCx/05BDV58PD+tdLSAAAAAAEAAQAABAcFoAAhADNAHA1SDCMeFlIYIhwZygDK7x8BHx4KFwYNBhJYBQEAP+0/Pz/dXe3tMwEQ1u0yENbtMDEBETY3NjMyFxYXFhURIxE0JyYjIgcGBxEjESM1MzUzFTMVAVtDVWFgkFBFGhS0HSuAW1g4Rauvr6u1BEr+5XJASEg+fV+h/doCVJVAX1o5df2ABEp43t54AAAAAAIAKAAAAikGtwADAAcAI0AMCQZEAwIECAcIBAIBugE9AAIBPAA/7T8/ARDWzs79xjAxEyEVIRczESMoAgH9/6S8vAa3eaD6YgAAAAACAAoAAAILBTQAAwAHACNADAkDAgVSBwgGBgUAAboBPQACAT4AP+0/PwEQ1v3Ozs4wMRMhFSEXMxEjCgIB/f+rrKwFNHmr+/AAAAAAAQBN/mgBiAWeABYAJkASGABEFRcVAhQICwYGBAQBAQAIAD8yETMRMy/NPz8BENb9xjAxISMGBhUUMzI3BwYjIiY1ND4CNyMRMwGITzoyRyVBBUdOQVIPKyAnArxGVy9cGVovWU0kO0EnKwWeAAAAAAIAOv5oAYQFlAANACQAMEAVJgkkUgIiJSQAGRQUEhIiDw8OBgUNuAE3AD/NPzMRMzMRMy/NPwEQ1jL9Ms4wMRImNTQ2MzIXFhUUBwYjEyMGBhUUMzI3BwYjIiY1ND4CNyMRM9xKSi8xJSMjJTFWOzoyRyVBBUdOQVIPKyAnBqwEpEYzMUYjITM1ISP7XEZXL1wZWi9ZTSQ7QScrBBAAAAACALAAAAGiBxAADQARACBADRMQRAsEDhIRCA4CAAi4ATYAP80/PwEQ1szM/cYwMQEyFxYVFAcGIyImNTQ2AzMRIwEpMSUjIyUxL0pKLry8BxAjITM1ISNGMzFG/o76YgABALUAAAFhBBAAAwAVQAkFAVIDBAIGAQAAPz8BENb9zjAxEzMRI7WsrAQQ+/AAAAACALT/IgO5BZ4AAwATACRAEhFEExUJDAJEABQSAgmXAggBAgA/Pz8/ARDW/cYyEN7hMDETMxEjARQOAgcmJic+AzURM7S8vAMFGjthRh0/HzZHKxO8BZ76YgFidaSAazwZOCA2UVx4XARUAAAEAIL+XwNfBaMAEwAnACsAOwA+QB89GTtSNCM5BSlSDys8OgA0NDGwKgYoACMeFKwPCgCsAD/NOT/NOT8/PzN8Lxg/ARDWMv0y1jLG/TLGMDETMh4CFRQOAiMiLgI1ND4CITIeAhUUDgIjIi4CNTQ+AgEzESMlFA4CByYmJz4DNREz9hgpHhISHykXGCogEhIgKgIQGCkeEhIfKRcYKiASEiAq/cisrAKlFzdZQBw6HTFDKBKsBaMSHioYFyogEhIfKhgYKh4SEh4qGBcqIBISHyoYGCoeEv5t+/BzbZh1YzcXMx0zTFVvVAOzAAIAuP2bBKUFtAAKABoAaUAoAgIBBAUEmwMCFAMCBAMcAgEACgIKAZsAChQACgEACkQHGgVEBxsAGbgBOUAMDrgJAgYICgUDCAADAD8/OTk/Pz/tAS8Q3v3MEO0QAMGHBSsQAMGHBX0QxAEYENYAwYcFK4cIfcQwM
```

Comment:
Not sure I’m a fan of the partial border (only around the text). Also, would be nice to simplify/modernize the logo. Maybe ditch the grid and play with the colors.